src/game/client/neo/ui/neo_hud_round_state.cpp

```diff
@@ -712,8 +719,124 @@ void CNEOHud_RoundState::DrawPlayerList()
 	}
 }
 
+void CNEOHud_RoundState::DrawPlayerList_BotCmdr()
+{
+	if (g_PR)
+	{
+		// Draw members of players squad in an old style list
+		const int localPlayerTeam = GetLocalPlayerTeam();
+		const int localPlayerIndex = GetLocalPlayerIndex();
+		const bool localPlayerSpec = !(localPlayerTeam == TEAM_JINRAI || localPlayerTeam == TEAM_NSF);
+		const int leftTeam = cl_neo_hud_team_swap_sides.GetBool() ? (localPlayerSpec ? TEAM_JINRAI : localPlayerTeam) : TEAM_JINRAI;
+
+		int offset = 52;
+		if (cl_neo_squad_hud_star_scale.GetFloat() > 0)
+		{
+			IntDim res = {};
+			surface()->GetScreenSize(res.w, res.h);
+			offset *= cl_neo_squad_hud_star_scale.GetFloat() * res.h / 1080.0f;
+		}
+
+		const bool hideDueToScoreboard = cl_neo_hud_scoreboard_hide_others.GetBool() && g_pNeoScoreBoard->IsVisible();
+
+		// Draw squad mates
+		m_commandedList.RemoveAll();
+		m_nonCommandedList.RemoveAll();
+		m_nonSquadList.RemoveAll();
+
+		bool squadMateFound = false;
+		m_iLeftPlayersAlive = 0;
+		m_iRightPlayersAlive = 0;
+		const int localStar = g_PR->GetStar(localPlayerIndex);
+
+		// Single pass to collect and categorize players
+		for (int i = 0; i < (MAX_PLAYERS + 1); i++)
```

Comment:
Player entity indices are one-indexed, so we could loop `i = 1; i <= MAX_PLAYERS` to skip the redundant i=0 case